src/styles/blocks/stars.css

```
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  display: flex;
+  align-items: center;
+  height: 16px;
+  width: 16px;
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
```

combine it all with comma
```suggestion
.stars--1 > .stars__star:nth-child(-n + 1),
.stars--2 > .stars__star:nth-child(-n + 2) {
  background-image: url(../../images/star-active.svg);
}

```

---

thanks, fixed